src/transformers/modeling_utils.py

```diff
@@ -892,7 +892,7 @@ def _get_resized_lm_head(
         if is_deepspeed_zero3_enabled():
             import deepspeed
 
-            with deepspeed.zero.GatheredParameters(old_lm_head.weight, modifier_rank=0):
+            with deepspeed.zero.GatheredParameters([old_lm_head.weight, old_lm_head.bias, new_lm_head.weight, new_lm_head.bias], modifier_rank=0):
```

Comment:
```suggestion
            params = [old_lm_head.weight, old_lm_head.bias, new_lm_head.weight, new_lm_head.bias]
            with deepspeed.zero.GatheredParameters(params, modifier_rank=0):
```
this would be more readable